### DISCUSSION TITLE
v2.227

### DISCUSSION THREAD

**Discussion by anddea** [anddea/revanced-patches#749] [Announcements]
The current version supports YouTube v19.16. Previously supported versions have been temporarily dropped. You may need to clear patch options to prevent potential errors.

If you have any questions, especially those not related to development, please consider contacting the community on [Telegram](https://t.me/AnddeaChat), Reddit, 4PDA, etc. for assistance first to save developers' time.

**Comment by Xisrr1** [comment]
Telegram link has been updated from https://t.me/AnddeaSupport to https://t.me/AnddeaChat.

**Reply by anddea** [reply]
Updated
